Setup.sh

```
@@ -4,10 +4,13 @@ set -e
 
 TEMPO_ROOT=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
+cd $TEMPO_ROOT
+
 if [ -z "$GIT_DIR" ]; then
 	GIT_DIR=$(git rev-parse --git-common-dir);
 	if [ $? -ne 0 ]; then
-		GIT_DIR=.git
+                echo "Failed to find .git folder"
```

We shouldn't hide this error